lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -76,188 +76,22 @@ def define
           if response == "none"
             outcome :outcome_not_affected
           else
-           question benefits.fetch(benefit_types.shift, :housing_benefit_amount?)
+            question benefits.fetch(benefit_types.shift, :housing_benefit_amount?)
           end
         end
       end
 
       #Q5a
-      money_question :bereavement_amount? do
+      benefits.each do |(benefit,method)|
```

Comment:
It's possible Rubocop will complain about the unused `benefit` parameter and the lack of space between `benefit` and `method`.